BUILDING.md

```diff
@@ -99,59 +99,50 @@ Node.js does not support a platform version if a vendor has expired support
-| macOS            | x64              | >= 10.15                        | Tier 1                                          | For notes about compilation see [^7]      |
-| macOS            | arm64            | >= 11                           | Tier 1                                          |                                           |
-| SmartOS          | x64              | >= 18                           | Tier 2                                          |                                           |
-| AIX              | ppc64be >=power8 | >= 7.2 TL04                     | Tier 2                                          |                                           |
-| FreeBSD          | x64              | >= 12.2                         | Experimental                                    |                                           |
-
-[^1]: GCC 8 is not provided on the base platform. Users will
-    need the
-    [Toolchain test builds PPA](https://launchpad.net/\~ubuntu-toolchain-r/+archive/ubuntu/test?field.series_filter=xenial)
-    or similar to source a newer compiler.
-
-[^2]: GCC 8 is not provided on the base platform. Users will
-    need the
-    [devtoolset-8](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-8/)
-    or later to source a newer compiler.
-
-[^3]: Older kernel versions may work for ARM64. However the
-    Node.js test infrastructure only tests >= 4.5.
-
-[^4]: On Windows, running Node.js in Windows terminal emulators
+| Operating System | Architectures    | Versions                          | Support Type                                    | Notes                                     |
+| ---------------- | ---------------- | --------------------------------- | ----------------------------------------------- | ----------------------------------------- |
+| GNU/Linux        | x64              | kernel >= 4.18[^1], glibc >= 2.28 | Tier 1                                          | e.g. Ubuntu 20.04, Debian 10, RHEL 8      |
+| GNU/Linux        | x64              | kernel >= 3.10, musl >= 1.1.19    | Experimental                                    | e.g. Alpine 3.8                           |
+| GNU/Linux        | x86              | kernel >= 3.10, glibc >= 2.17     | Experimental                                    | Downgraded as of Node.js 10               |
+| GNU/Linux        | arm64            | kernel >= 4.18[^1], glibc >= 2.28 | Tier 1                                          | e.g. Ubuntu 20.04, Debian 10, RHEL 8      |
+| GNU/Linux        | armv7            | kernel >= 4.18[^1], glibc >= 2.28 | Tier 1                                          | e.g. Ubuntu 20.04, Debian 10              |
+| GNU/Linux        | armv6            | kernel >= 4.14, glibc >= 2.24     | Experimental                                    | Downgraded as of Node.js 12               |
```

Comment:
I do wonder about this one as well, but I think it falls into the same category of being part of unofficial-builds. It does seems strange though.

Reply:
Yeah, this one's also under unofficial-builds.